game/src/sandbox/dashboards/risks.rs

```diff
@@ -26,12 +28,33 @@ impl RiskSummaries {
             modes: maplit::btreeset! { TripMode::Bike },
             include_no_changes,
         };
-
         let ped_filter = Filter {
             modes: maplit::btreeset! { TripMode::Walk },
             include_no_changes,
         };
 
+        let (arterial_problems, mut trip_lookup) = problem_matrix(
```

Comment:
All of this extra code is necessary to build up `trip_lookup` here, so we can match up a made-up string from `Outcome::Clicked` to a list of trips. It would be much more straightforward to make the widget directly store and return the `Vec<TripID>`. I made a very old attempt in https://github.com/a-b-street/abstreet/commit/737ec428527f4a683b0ddf4584359dec98ccfc4b to make `Outcome` generic, but I think I'm going to try again using `any` / casting.